platform/xbox/tls.c

```diff
@@ -1,6 +1,8 @@
 #include <xboxkrnl/xboxkrnl.h>
 #include <pdclib/_PDCLIB_xbox_tls.h>
 
+#pragma comment(linker, "/include:__tls_index")
+
```

Comment:
This blank line probably shouldn't be here (also not part of similar https://github.com/XboxDev/nxdk/pull/298/files)

Reply:
Changed.